README.md

```diff
@@ -13,44 +15,44 @@ After installing the template, you can quickly create the following items:
 * masafx-service: MASA Framework Pure Template (Minimal Api)
 * masafx-service-cqrs: MASA Framework CQRS Template(Minimal Api、CQRS)
 
-## Install Or Upgrade Template
-
-open windows terminal and run commands:
+## Install template
 
 ```shell
 dotnet new install Masa.Template
 ```
 
-## Get Version
-
-open windows terminal and run commands:
+### Install specific version or prerelease version
 
 ```shell
-dotnet new uninstall
+dotnet new install Masa.Template::1.0.0-rc.1
 ```
 
-## Uninstall Template
-
-open windows terminal and run commands:
+## Uninstall template
 
 ```shell
 dotnet new uninstall Masa.Template
 ```
 
 ## How to use
 
-Take the masa framework template as an example, open windows terminal and run commands:
+Take the masa framework template(masafx) as an example, run commands:
 
-### Get Help
+### Get help
 
 ```shell
 dotnet new masafx -h
 ```
 
-> dotnet new {templateName} -h
-
-### Create Project
+### Create project
 
 ```shell
 dotnet new masafx -n Masa.EShop
 ```
+
+## Develop or install locally
+
+```shell
+git clone https://github.com/masastack/MASA.Template.git
+cd MASA.Template
+dotnet new install .\src\ --force
```

Comment:
Add a command to uninstall the local version?

Reply:
fine.